### PR TITLE
Expand tested Python versions

### PR DESCRIPTION
**Pull request by ajnelson-nist**

### CODE REVIEW

**Comment by ajnelson-nist** [comment]
I'll merge this once CI passes.

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -12,7 +12,9 @@ jobs:
     strategy:
       matrix:
         os: ['ubuntu-latest', 'macos-latest']
-        python-version: [3.8, 3.9]
+        python-version:
+          - '3.8'
+          - '3.12'
     steps:
```
Comment (simsong) [inline .github/workflows/continuous-integration.yml:18]: yay!

**Review by simsong — COMMENTED** [review]
great.

**Review by simsong — COMMENTED** [review]
won't let me approve?

**Comment by ajnelson-nist** [comment]
> won't let me approve?

Sorry, I just went through with merging after CI passed, because you had suggested this change in #46.

**Pull request merged**